<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SScanWidget</class>
 <widget class="QWidget" name="SScanWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SScanWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QTableWidget" name="TableModule">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="maximumSize">
        <size>
         <width>360</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="Placehold1">
          <property name="font">
           <font>
            <family>Sarasa UI SC</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frameBase">
          <property name="font">
           <font>
            <family>Sarasa UI SC</family>
           </font>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QRadioButton" name="Base10">
             <property name="font">
              <font>
               <family>Sarasa UI SC</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>10进制</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Base16">
             <property name="font">
              <font>
               <family>Sarasa UI SC</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>16进制</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Base2">
             <property name="font">
              <font>
               <family>Sarasa UI SC</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>2进制</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="ExcludeCheckbox">
             <property name="font">
              <font>
               <family>Sarasa UI SC</family>
              </font>
             </property>
             <property name="statusTip">
              <string>扫描时排除指定的值</string>
             </property>
             <property name="text">
              <string>排除</string>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="ScanValueLabel">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sarasa UI SC</family>
           </font>
          </property>
          <property name="text">
           <string>数值:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="TargetValueA">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Sarasa UI SC</family>
              </font>
             </property>
             <property name="maxLength">
              <number>999999999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="TargetValueB">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Sarasa UI SC</family>
              </font>
             </property>
             <property name="maxLength">
              <number>999999999</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sarasa UI SC</family>
           </font>
          </property>
          <property name="text">
           <string>数值类型:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QFrame" name="frame2">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="FindType">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Sarasa UI SC</family>
              </font>
             </property>
             <property name="placeholderText">
              <string>数值类型</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="FindMethod">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Sarasa UI SC</family>
              </font>
             </property>
             <property name="placeholderText">
              <string>扫描方式</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QFrame" name="frameButton">
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="0" column="1">
            <widget class="QPushButton" name="ButtonRestart">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Sarasa UI SC</family>
              </font>
             </property>
             <property name="text">
              <string>重新开始</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="ButtonSearch">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Sarasa UI SC</family>
              </font>
             </property>
             <property name="text">
              <string>扫描</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="ButtonUndo">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Sarasa UI SC</family>
              </font>
             </property>
             <property name="text">
              <string>撤销</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="TableFound">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ContiguousSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QProgressBar" name="ScanProgress">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>6</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="qLabel">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>结果:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="FoundCount">
       <property name="font">
        <font>
         <family>Sarasa Mono SC</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
